### ISSUE TITLE
[Bug] Dropzone field throws UnableToRetrieveMetadata exception if files are missing: Unable to retrieve the file_size for file at location

### ISSUE THREAD

**Issue by dandarie** [Laravel-Backpack/CRUD#5377]
# Bug report

### What I did

Used Dropzone as intended: https://backpackforlaravel.com/docs/6.x/crud-fields#dropzone-pro

Then I tested what would happen if the files are missing (external intervention, storage malfunctions).

### What I expected to happen

The dropzone field should either ignore the missing files or show them as defective (placeholder) and offer a way to remove them.

### What happened

UnableToRetrieveMetadata is thown with message "Unable to retrieve the file_size for file at location: [file_path]"

### What I've already tried to fix it

In vendor/backpack/pro/resources/views/fields/dropzone.blade.php, replace from line 25:

```
            $serverFiles[] = [
                'name' => basename($path),
                'size' => \Storage::disk($disk)->size($path),
                'mime' => \Storage::disk($disk)->mimeType($path),
                'path' => $path,
                'url' => Storage::disk($disk)->url($path),
            ];
```

with:

```
            try {
                $serverFiles[] = [
                    'name' => basename($path),
                    'size' => \Storage::disk($disk)->size($path),
                    'mime' => \Storage::disk($disk)->mimeType($path),
                    'path' => $path,
                    'url' => Storage::disk($disk)->url($path),
                ];
            } catch (\League\Flysystem\UnableToRetrieveMetadata $e) {
                $serverFiles[] = [
                    'name' => basename($path),
                    'size' => -1,
                    'mime' => '',
                    'path' => $path,
                    'url' => Storage::disk($disk)->url($path),
                ];
            }
```

### Is it a bug in the latest version of Backpack?

Yes

### PHP VERSION:
```
PHP 8.1.21 (cli) (built: Jul  8 2023 07:10:19) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.21, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.21, Copyright (c), by Zend Technologies
    with Xdebug v3.2.1, Copyright (c) 2002-2023, by Derick Rethans
```

### LARAVEL VERSION:
`10.33.0.0`

### BACKPACK PACKAGE VERSIONS:
```
backpack/basset: 1.2.2
backpack/crud: 6.3.0
backpack/filemanager: 3.0.6
backpack/generators: v4.0.2
backpack/medialibrary-uploaders: 1.0.1
backpack/pro: 2.0.18
backpack/theme-coreuiv4: 1.1.1
backpack/theme-tabler: 1.1.1
```

**Comment by pxpm** [comment]
It's in the private repo @dandarie 

I will have a look at this during the weekend, probably Sunday as I think I will not be able to work this out today/tomorrow. 

I am still not sure what's the best way to deal with this. Do we delete the file (string in the files array) ? What if it was just a temporary error ?

I need to spend a bit on this, I will get back to you with news 🙏 

Cheers and thanks for the report.

**Comment by pxpm** [comment]
Hey @dandarie sorry it took me a while to get back here. 
I've just released `Backpack\PRO 2.0.19` with the fix for this issue. 

It should take a couple hours for our private repo to catch up you should then be able to get the fix with a `composer update backpack/pro`. 

Thanks again.
